metagraph/src/cli/config/config.hpp

```diff
@@ -91,6 +91,7 @@ class Config {
     unsigned int genome_binsize_anno = 1000;
     unsigned int arity_brwt = 2;
     unsigned int relax_arity_brwt = 10;
+    unsigned long long row_sparse_disk_buff_size = 10'000; // in B
```

Comment:
why not make this more generic and call it `disk_buffer_size`?

Reply:
Answered above